templates/github/.github/workflows/release.yml.j2

```diff
@@ -32,7 +32,7 @@ jobs:
 
       - uses: actions/setup-python@v2
         with:
-          python-version: "3.8"
+          python-version: "{{ python_version }}"
```

Comment:
I think i remember, we discussed this. And the release workflow does not run any pulp code so we opted out from making the python version flexible.
Am i right here? @daviddavis

Reply:
That's basically correct except I believe there was release Python code that didn't wasn't compatible with older versions of Python.

Reply:
The release code does use `python_version` when running tests:

https://github.com/pulp/plugin_template/pull/474/files#diff-2804d6731049ff36e9d9c9de50d1541440740aa9fd2479a75714f4e1bfc361ddL93